multinet/auth/google.py

```diff
@@ -91,10 +98,13 @@ def init_oauth(app: Flask) -> None:
 @bp.route("/login")
 @use_kwargs({"return_url": fields.Str(location="query")})
 @swag_from("swagger/google/login.yaml")
-def login(return_url: str) -> ResponseWrapper:
+def login(return_url: str = None) -> ResponseWrapper:
```

Comment:
I think the type of `return_url` needs to be updated to `Optional[str]`.

Reply:
Good catch, not really sure why mypy doesn't catch that :thinking:. Might be due to the decorators.